WooCommerce/src/main/kotlin/com/woocommerce/android/ui/themes/ThemePickerViewModel.kt

```diff
@@ -28,11 +28,14 @@ class ThemePickerViewModel @Inject constructor(
     private val themeRepository: ThemeRepository,
     private val resourceProvider: ResourceProvider,
 ) : ScopedViewModel(savedStateHandle) {
-    private val _viewState = savedStateHandle.getStateFlow<ViewState>(viewModelScope, Loading)
-    val viewState = _viewState.asLiveData()
-
     private val navArgs: ThemePickerFragmentArgs by savedStateHandle.navArgs()
 
+    private val _viewState = savedStateHandle.getStateFlow(
+        scope = viewModelScope,
+        initialValue = ViewState(navArgs.isFromStoreCreation, Loading)
```

Comment:
np: I wonder if it helps code readability to name the arguments. Especially useful for `isSkipButtonVisible`, I think, to clarify why it matters to pass `isFromStoreCreation` for it:

```suggestion
        initialValue = ViewState(isSkipButtonVisible = navArgs.isFromStoreCreation, carouselState = Loading)
```

Reply:
Yeah, that's a good idea 👍